README.md

```diff
@@ -1,2 +1,66 @@
-# tmail-ui-user
-This project is a user interface Flutter app for [TMail back-end](https://github.com/linagora/tmail-backend)
+# Tmail Flutter mobile application
+
+![wider](https://user-images.githubusercontent.com/6928740/129552343-31e21182-d07c-4f2a-bc83-4f9be70d1693.png)
+
+This project aims at providing a multi-plateform mobile email application, running the [JMAP protocol](https://jmap.io/) and will also deliver additional 
+features to the [TMail back-end](https://github.com/linagora/tmail-backend).
+
+## FAQ
+
+ - **Why did you choose JMAP?**
+
+That is a good question! **IMAP** is THE ubiquitous protocol people use to read their emails, THE norm.
+
+Yet **IMAP** had been designed in another age, which resulted in a chatty patchwork of extensions. **IMAP** lacks decent synchronisation primitives to address real-time challenges modern mobile fleet requires, it consumes a lot of bandwith, requires a lot of roundtrips which means high latency.
+
+We are not alone to say this! Big player of the field started their own [proprietary](https://developers.google.com/gmail/api) [protocols](https://docs.microsoft.com/en-us/exchange/clients/exchange-activesync/exchange-activesync?view=exchserver-2019) to address IMAP flaws. Yet, and inter-operable standard was yet to be found... 
+
+This, is where **[JMAP](https://jmap.io/)** comes to play! **JMAP** builds on decades of experience, and beautifully addresses these chalenges using `HTTP` and `JSON` standards. Not only does it makes applications easier to develop, we also strongly believes it results in an improved experience for the end user.
+
+### **Can I use TMail with *any* JMAP server?**
+
+Yes, you can use the TMail application with any JMAP server and benefits from **Tmail** ergonomy and ease of use.
+
+### **I don't understand your crap... I need help using your App! HELP MEEEEEE...**
+
+Don't worry, we are here!
+
+We plan on writing a user documentation, helping you navigating around the application, and detailing the few configurations you have to perform.
+
+If what you are looking for is not in the *user guide* then ask us directly in the [issues](https://github.com/linagora/tmail-flutter/issues), we would first be glad to help. But also glad to improve our documentation and may tweak slightly our UI (user interface).
+
+### **What plateforms do you (plan to) target?**
+
+First, we target Android, IOS mobiles. We also take care of tablets and large rendering space early on in the development process.
+
+Then, we plan on introducing a desktop applications.
+
+This versatility is enabled by the use of the [Flutter framework](https://flutter.dev/).
+
+### **What would your roadmap look like?**
+
+First, we plan to write a simple, multi-plateform JMAP email client. This includes reading your mail and mailboxes, managing them, sending emails, searching your emails. This will likely keep us busy by the end of 2021.
```

Comment:
```suggestion
First, we plan to write a simple, multi-plateform JMAP email client. This includes reading your mails and mailboxes, managing them, sending emails, searching your emails. This will likely keep us busy by the end of 2021.
```